runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -608,7 +608,7 @@ class WorkItemListPage {
 
   /* Iterations Page object model */
   clickIterationKebab (index){
-    return element(by.xpath ("(.//button[@type='button'])["+ index +"]")).click();
+    return element(by.xpath ("(.//button[@class='btn btn-link dropdown-toggle']/i[@class='fa fa-ellipsis-v'])["+ index +"]")).click();
```

Comment:
This is fine for the short-term, but can you see about adding a unique ID for the button? Thx!